sqlmesh/core/config/common.py

```diff
@@ -1,12 +1,31 @@
 from __future__ import annotations
 
 import typing as t
+from enum import Enum
 
 from pydantic import validator
 
+from sqlmesh.utils import classproperty
 from sqlmesh.utils.errors import ConfigError
 
 
+class EnvironmentSuffixTarget(str, Enum):
```

Comment:
this class seems unnecessary

Reply:
The alternative is doing a bool. So someone would pass in `use_schema_for_environment_suffix: True`. ON this one they provide `environment_suffix_target: schema` (or `table`). Not really a big difference and this has the nice property that it can be expanded in the future (maybe `catalog` later?).

Reply:
i'd prefer a bool for now

Reply:
Within the code is nearly functions the same so I don't see much of a reason there. For users if you set `use_schema_for_environment_suffix: False` then what does it do instead? It is not obvious. Setting `environment_suffix_target: table` is now very clear. So I just don't see much upside to the bool so I'm going to leave it unless someone wants to strongly advocate for bool.

Reply:
ok